src/app/state/rest/tasks.ts

```diff
@@ -169,3 +169,15 @@ export const useTaskNames = (role: string) => {
     isProtected: true
   })
 }
+
+// useSummonTypes for getting the available summonTypes for a specific task
+export const useSummonTypesByTaskId = (id: number | string, options?: Options) => {
```

Comment:
id kan een nummer of string zijn, is dat wel juist?

Reply:
Yes, you're right!